docs/configuration.rst

```diff
@@ -971,3 +971,100 @@ This must be used in conjunction with ``encrypted = true`` and must have a custo
 See `Encrypted EBS with a Custom KMS Key <_encrypted_ebs>`. ::
 
     ebs_kms_key_id = xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+
+
+.. _fsx_section:
+
+FSx
+^^^
+Configuration for an attached FSx Lustre file system.
+
+Create a new file system, by omitting ``fsx_fs_id``, or use an existing one by specifying ``fsx_fs_id``. ::
+
+    [fsx fs]
+    shared_dir = /fsx
+    fsx_fs_id = fs-073c3803dca3e28a6
+    storage_capacity = 3600
+    fsx_kms_key_id = 9e8a129c-0e85-459-865b-3a5be974a22b
+    imported_file_chunk_size = 1024
+    export_path = s3://bucket/folder
+    import_path = s3://bucket
+    weekly_maintenance_start_time = 1:00:00
+
+shared_dir
+""""""""""
+**Required** Defines the mount point for the Lustre File system on the master and compute nodes.
+
+The example below will mount the filesystem at /fsx.
+
+Do not use NONE or /NONE as the shared directory.::
```

Comment:
Side question: do we validate this?

Reply:
That it's not `NONE` or `/NONE`? We don't.